HW0/P4/P4.py

```diff
@@ -30,47 +30,97 @@
     # Load true trajectory and plot it
     # Normally, this data wouldn't be available in the real world
     #####################
+    s_true=np.loadtxt('P4_trajectory.txt',delimiter=',')
```

Comment:
You can also use pandas to load information from text files; I prefer that. But what you did is great too!